faq.md

```diff
@@ -3,3 +3,27 @@ layout: page
 title: FAQ
 permalink: /faq
 ---
+# How can I help?
+
+If you have a service like this for people already, get in contact with us at [contact@vacfind.org][1]!
+We are commited to bringing easy to access information such as where to get a vaccine to the general public.
+Have some spare time on your hands?
+[Email us][1]!
+
+# How can Hospitals help?
+
+Hospitals and clincs administering vaccines are our end targets.
+We can add your data to our site and help bring people to you.
+It will also relieve your databases and phone calls as we can provide this directly to your customers.
+All we need is up to date information in whatever form you find easiest.
+
+# Don't have time?
+
+Don't worry! You can also [donate to us][2].
+
+# Contact Us
+
+To get into contact, email us at [contact@vacfind.org][1].
```

Comment:
lets try and use the jekyll variable wherever we can in case anyone forks our site and wants to easily rebrand it for their org